src/main/java/uk/gov/justice/laa/amend/claim/controllers/ChangeMonetaryValueController.java

```
@@ -42,7 +42,12 @@ public String getMonetaryValue(
         try {
             ClaimDetails claim = (ClaimDetails) request.getAttribute(claimId);
             ClaimField claimField = cost.getAccessor().get(claim);
-            BigDecimal value = claimField != null ? (BigDecimal) claimField.getAmended() : null;
+
+            if (claimField == null) {
+                return String.format("redirect:/submissions/%s/claims/%s", submissionId, claimId);
+            }
```

Likewise here, should we throw an error or redirect?

---

Sure. Will throw an error.